openslides_backend/action/actions/meeting/update.py

```diff
@@ -0,0 +1,149 @@
+    "motions_statute_recommendations_by",
+    "motions_recommendation_text_mode",
+    "motions_default_sorting",
+    "motions_number_type",
+    "motions_number_min_digits",
+    "motions_number_with_blank",
+    "motions_statutes_enabled",
+    "motions_amendments_enabled",
+    "motions_amendments_in_main_list",
+    "motions_amendments_of_amendments",
+    "motions_amendments_prefix",
+    "motions_amendments_text_mode",
+    "motions_amendments_multiple_paragraphs",
+    "motions_supporters_min_amount",
+    "motions_export_title",
+    "motions_export_preamble",
+    "motions_export_submitter_recommendation",
+    "motions_export_follow_recommendation",
+    "motion_poll_ballot_paper_selection",
+    "motion_poll_ballot_paper_number",
+    "motion_poll_default_type",
+    "motion_poll_default_100_percent_base",
+    "motion_poll_default_majority_method",
+    "motion_poll_default_group_ids",
+    "users_sort_by",
+    "users_enable_presence_view",
+    "users_enable_vote_weight",
+    "users_allow_self_set_present",
+    "users_pdf_welcometitle",
+    "users_pdf_welcometext",
+    "users_pdf_url",
+    "users_pdf_wlan_ssid",
+    "users_pdf_wlan_password",
+    "users_pdf_wlan_encryption",
+    "users_email_sender",
+    "users_email_replyto",
+    "users_email_subject",
+    "users_email_body",
+    "assignments_export_title",
+    "assignments_export_preamble",
+    "assignment_poll_ballot_paper_selection",
+    "assignment_poll_ballot_paper_number",
+    "assignment_poll_add_candidates_to_list_of_speakers",
+    "assignment_poll_sort_poll_result_by_votes",
+    "assignment_poll_default_type",
+    "assignment_poll_default_method",
+    "assignment_poll_default_100_percent_base",
+    "assignment_poll_default_majority_method",
+    "assignment_poll_default_group_ids",
+]
+
+
+@register_action("meeting.update")
+class MeetingUpdate(UpdateAction):
+    model = Meeting()
+    schema = DefaultSchema(Meeting()).get_update_schema(
+        optional_properties=[
+            *meeting_settings_keys,
+            "template_for_committee_id",
+            "guest_ids",
+            "reference_projector_id",
+        ],
+        additional_optional_fields={
+            "default_projector_$_id": used_as_default_for_schema,
+        },
+    )
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        meeting_check = []
+        if "reference_projector_id" in instance:
+            if reference_projector_id := instance["reference_projector_id"]:
+                meeting_check.append(
+                    FullQualifiedId(Collection("projector"), reference_projector_id)
+                )
+            else:
+                raise ActionException("The reference projector can't be set to null.")
+        if "default_projector_$_id" in instance:
+            if any((id is None for id in instance["default_projector_$_id"].values())):
+                raise ActionException("The default projecor can't be set to null.")
```

Comment:
I would suggest to set this relation to `required` in the models file, so you can save these checks.

Reply:
If we set them to required:
- the reference_projector_id required will be caught by the schema validator
- the default_projector_$_id required can be caught also by the schema validator. We have to create a required validation schema
- To catch these errors in the validate_required fields method would be more difficult:
-- it's difficult to bypass the schema and other validations
-- Template fields are not implemented on required. We had to implement this with regex for only this case

I think we should realize the schema validation variant